pkg/deploy/lattice/rule_manager.go

```diff
@@ -360,8 +359,13 @@ func isRulesSame(modelRule *latticemodel.Rule, sdkRuleDetail *vpclattice.GetRule
 		}
 	}
 
-	// Header Match
+	// Method match
+	if aws.StringValue(sdkRuleDetail.Match.HttpMatch.Method) != modelRule.Spec.Method {
+		glog.V(6).Infof("Method mismatch '%v' != '%v'\n", modelRule.Spec.Method, sdkRuleDetail.Match.HttpMatch.Method)
```

Comment:
The way of logging will change pretty soon anyways but for the note, trailing `\n` is not required - also I would recommend avoiding use of `%v` if possible.

Reply:
So far I've tried to stick with whatever convention was in the file. Happy to follow up on these as part of the ongoing logging improvements.